src/main/java/com/entropyteam/entropay/employees/models/Gender.java

```diff
@@ -0,0 +1,7 @@
+package com.entropyteam.entropay.employees.models;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+@JsonFormat(shape = JsonFormat.Shape.STRING)
+public enum Gender{MALE, FEMALE, NO_BINARY}
```

Comment:
one last and small change, it should be NON_BINARY right?

Reply:
Had exactly the same doubt last week but in the Basecamp ticket said "Male, Female, No Binary" and I took that for reference to create the enum. Anyway, I changed it and now it's NON_BINARY